src/test/java/io/outfoxx/swiftpoet/test/AttributeSpecTests.kt

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2018 Outfox, Inc.
```

Comment:
```suggestion
 * Copyright 2023 Outfox, Inc.
```

Reply:
These fail a lint check when I tried it also with 2023.

Reply:
I think its powered by `HEADER.txt` which expects 2018 to be there.

Reply:
Seems like not supported at this time https://github.com/CadixDev/licenser/issues/11